packages/devtools-launchpad/src/components/LandingPage.css

```diff
@@ -93,6 +93,10 @@
   transition: var(--base-transition);
 }
 
+.landing-page .panel header input[type=button] {
+  flex: 0.25;
```

Comment:
urgh, sorry about that
Only the search input should have the `flex: 1` property, so we may target it specifically with `input[type="search"]`